java/build.xml

```diff
@@ -45,14 +45,14 @@
     </exec>
   </target>
 
-  <target name="build-phone-metadata-to-one-file">
+  <target name="build-single-file-phone-metadata">
```

Comment:
Would it make sense to rename the other rule too, e.g., to "build-multi-file-phone-metadata"? And then have a comment above them both that exactly one of them should be used? (Same comment re: test targets.)